doc/diffs.md

```diff
@@ -68,53 +166,184 @@ The "jit-diff list" command has this help message:
 
-The jit-analyze tool understands the format of the `*.dasm` files in a diff and can extract
-this data to produce a summary and/or dump the info to a data file (in CSV or JSON format).
-The common usage of the tool is to extract interesting diffs, if any, from a diff run as
-part of the development progresses.
+jit-analyze will be run to compare the generated output.
 
-This tool can be invoked automatically after generating diffs using the `-a` option to `jit-diff diff`.
+You can also run a recursive textual comparison tool like windiff on Windows to visually compare
+the diffs, e.g.:
+```
+    c:\gh\coreclr> windiff c:\diffs\dasmset_12\base c:\diffs\dasmset_12\diff
+```
 
-Sample help command line:
+## Simplified jit-diff usage: defaults
+
+As seen above, specifying all required information can be quite verbose. jit-diff can automatically determine
+most of the arguments using computed defaults. The above diff can be accomplished using simply:
+```
+    c:\gh\coreclr> jit-diff diff --base_root e:\gh\coreclr2
+```
+
+The defaults are:
+* If jit-diff is invoked with the current directory within the dotnet/coreclr repo, then the root of
+  this repo serves to find the diff compiler and `CORE_ROOT` directory. (Note that we have no reasonable
+  default for determining what the baseline toolset or repo is. This is specified with the `--base_root` argument.)
+* The default output directory is `<repo_root>\diffs` (in this case, c:\gh\coreclr\diffs).
+* The default diff architecture is x64. If this isn't found, x86 is tried.
+* The default diff build flavor is checked. If this isn't found, debug is tried.
+* By default, diffs are done using System.Private.CoreLib.dll. (That is, `--corelib` is the default.)
+
+To instead do diffs over the framework assemblies (not just System.Private.CoreLib.dll), using an x86
+debug build, run:
+```
+    c:\gh\coreclr> jit-diff diff --frameworks --arch x86 --build debug --base_root e:\gh\coreclr2
+```
+
+To simplify this more, create a dotnet/coreclr repo clone that you will always use for baselines.
+Create a configuration file that specifies your baseline root directory, specifying a `base_root`
+default. See the document [configuring defaults](config.md) for details.
+
+With a `--base_root` default in the config.json file, you can simply run:
+```
+    c:\gh\coreclr> jit-diff diff
+```
+to generate diffs using all the defaults.
+
+The following command-line argument are used to adjust the defaults, such as specifying x86 diffs
+instead of x64 diffs. They are not otherwise required.
+* `--base_root`
+* `--diff_root`
+* `--arch`
+* `--build`
+
+## Using tags
+
+jit-diff takes an optional '--tag' command-line argument.  This tag can be used to label different
+directories of `*.dasm` in the output directory so multiple runs can be done.
+This supports a scenario like the following:
+
+* Build base CoreCLR
+* Produce baseline diffs by invoking the tool with '--base'
```

Comment:
Yeah, I guess there's a usage problem here where the defaults will kick in and find the diff compiler. Maybe there should be a way to prevent this. Maybe if you specify --base or --diff directly, the defaults don't kick in.